rest-client/swagger.json

```diff
@@ -162,6 +162,21 @@
       "in": "query",
       "type": "string"
     },
+    "pageSize": {
+      "name": "pageSize",
+      "description": "maximum number of records in each returned page",
```

Comment:
Looks good but we should add the default and the max value to the documentation.